bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
 
         # create new chart options to get default properties with values
         default_props = ChartOptions().properties_with_values()
 
         # get the properties and values from the module-wide defaults object
         option_props = ChartOptions.properties_with_values(defaults)
```

Comment:
Here's another idea for how to do this:
 * if people want per-module customization of `ChartOptions`, they can do `default_options.apply_theme()`
 * in `Chart` constructor, do `self._options.apply_theme(default_options.themed_values())` which will apply the themed values between the defaults and the kwarg values

Or if `apply_theme` is awkward and direct assignment to `default_options` would be cleaner, we could do:
 * if people want per-module customization of `ChartOptions`, they assign to properties of `default_options`
 * in `Chart` constructor, do `self._options.apply_theme(default_options.properties_with_values(include_defaults=False))` which will apply the changed values from `default_options` between the defaults and the kwarg values